### PR TITLE
chore(setup-react-router-dom) Add router-dom

### PR DESCRIPTION
**Pull request by kelvinndmo**
**What does this PR do?**

- Setting up react-router-dom

**Description of Tasks to be completed**

- Setup react-router-dom
- Testing the functionality with a sample login page

**How should this be manually tested?**

- Clone this repository then navigate to `ft-setup-react-router-164047019`
- Run `npm install` to install dependencies
- Run `npm start` to run the application.
- Change url to `....\login`.It should change without any errors

### CODE REVIEW

**Review by mnswaleh — DISMISSED** [review]

**Comment by kevpy** [comment]
The changes are okay. The PR is good to me.

**Review by winniekariuki — DISMISSED** [review]

**Comment by winniekariuki** [comment]
Looks good to me

**Review by kevpy — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -13,7 +13,9 @@
   "jest": {
     "snapshotSerializers": [
       "enzyme-to-json/serializer"
-    ]
+    ],
```
Comment (Derrickkip) [inline package.json]: These packages are in the wrong place

**Review by Derrickkip — CHANGES_REQUESTED** [review]
Check the package.json the dependancies you added are in the wrong place

**Review by Kibetchirchir — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (kelvinndmo) [reply package.json]: Thanks for noting that.Worked on it.

**Review by kevpy — APPROVED** [review]

**Review by mnswaleh — APPROVED** [review]

**Review by Derrickkip — APPROVED** [review]

**Pull request merged**